Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.Features, "Local Settings", Description = "Use settings to remember the user-customizable aspects of your app. For example, a news reader could use app settings to save which news sources to display and what font to use for reading articles.", DocumentationLink = "https://learn.microsoft.com/en-us/windows/uwp/get-started/settings-learning-track")]
```

Comment:
If https://github.com/unoplatform/Uno.Gallery/pull/514 merges first, we can change this to `SampleCategory.NonUIFeatures`

Reply:
Merge master und switched to NonUIFeatures